src/Http/Response/PagedCollectionResponse.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Response;
+
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class PagedCollectionResponse extends JsonResponse
+{
+    public function __construct(
+        int $itemsPerPage,
+        int $totalItems,
+        array $members = [],
```

Comment:
I'm not a big fan of `member` here as a variable name. It is supposed to mean `items`?

Reply:
It's supposed to reflect the `member` key in the response object. I'm going to merge this because I'm pair programming on something else but Corneel is waiting for this PR to deploy.